index.html

```diff
@@ -1070,6 +1082,9 @@ <h3>Data delivery</h3>
                     <li>
                       Run [=queue a record=] with |observer|, |source|, |record|.
                     </li>
+                    <li>
+                      Clear |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|].
```

Comment:
That would in my mind be removing all entries in the ordered map (or assigning it to an empty tuple).

It is also used in HTML and defined in infra:

![image](https://github.com/w3c/compute-pressure/assets/1101339/3c93ab45-228b-447e-8981-93f8b96a7dd5)

@arskama you should link to the definition: https://infra.spec.whatwg.org/#map-clear

Reply:
taking is not defined (it is used as remove from ordered map and keep a reference to it)

Should probably to rewritten to those two steps instead

Reply:
This would be the proposed fix.
Because "Taking" is not an existing term, it can be replaced by read and remove.
"clear" was really a misused word here. The idea was to remove the entry at a specific [source] key.

@@ -1077,13 +1071,13 @@ of system resources such as the CPU.
                   Create timer of |observer|.{{PressureObserver/[[PenaltyDuration]]}} duration with the following callback:
                   <ol>
                     <li>
-                      Let |record| be the result of taking |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|].
+                      Let |record| be |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|].
                     </li>
                     <li>
                       Run [=queue a record=] with |observer|, |source|, |record|.
                     </li>
                     <li>
-                      Clear |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|].
+                      Remove |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[|source|].
                     </li>
                   </ol>
                 </li>